requirements.txt

```diff
@@ -1,8 +1,8 @@
-llvmlite==0.41.1
-numba==0.58.1
-numpy==1.26.2
-pandas==2.1.4
-python-dateutil==2.8.2
-pytz==2023.3.post1
-six==1.16.0
-tzdata==2023.3
+llvmlite==0.43.0
+numba==0.60.0
+numpy==2.0.2
+pandas==2.2.3
+python-dateutil==2.9.0.post0
```

Comment:
Seems kind of weird to me that they used this non-standard version suffix instead of just cutting `2.9.1`, but it's indeed [the most recent public version of dateutil](https://github.com/dateutil/dateutil/releases/tag/2.9.0.post0).